src/PowerSimulations.jl

```diff
@@ -329,17 +329,7 @@ import LinearAlgebra
 import JSON3
 import PowerSystems
 import InfrastructureSystems
-# so that users have access to IS.Results interfaces
-import InfrastructureSystems:
-    get_variables,
-    get_parameters,
-    get_total_cost,
-    write_results,
-    get_timestamp,
-    get_resolution,
-    get_name,
-    @assert_op,
-    list_recorder_events
+import InfrastructureSystems: @assert_op, list_recorder_events
 export get_name
```

Comment:
Is this now a problem?

Reply:
> Is this now a problem?

Maybe not. It is implemented on OperationalModel. Do you want that method to be an IS/PSY method? If someone ever `using PowerSimulations, PowerSystems`, there would be a conflict because of this change.